dev/devicelab/lib/framework/ab.dart

```diff
@@ -30,6 +32,48 @@ class ABTest {
     _addResult(result, _bResults);
   }
 
+  static void accumulateLengths(List<int> lengths, List<String> results) {
```

Comment:
`accumulateLengths` sounds like that we're summing up lengths. Maybe `updateColumnLengths` is a better name?